src/NGen.cpp

```diff
@@ -69,6 +69,8 @@ int main(int argc, char *argv[]) {
               << ngen_VERSION_PATCH << std::endl;
     std::ios::sync_with_stdio(false);
 
+    ut_set_error_message_handler(ut_ignore);
```

Comment:
need to remove this.  If you want to disable unit messages, build with `-DUDUNITS_QUIET`

Reply:
I think that is a commit that somehow got from one of my profiling branches into my main and has been rebased onto code sense then.

Reply:
Any it is removed